protocol/contracts/v2/TreasuryReservesVault.sol

```diff
@@ -185,6 +185,7 @@ contract TreasuryReservesVault is ITreasuryReservesVault, TempleElevatedAccess {
         uint256 _length = debtCeiling.length;
         for (uint256 i; i < _length; ++i) {
```

Comment:
```
unchecked {
  ++i;
}
```

Reply:
not worth the bad readability imo given this won't be a large list and won't be called that often.